feature-libs/checkout/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@spartacus/checkout",
+  "version": "3.0.0",
+  "description": "Checkout feature library for Spartacus",
+  "keywords": [
+    "spartacus",
+    "framework",
+    "storefront",
+    "checkout"
+  ],
+  "homepage": "https://github.com/SAP/spartacus",
+  "repository": "https://github.com/SAP/spartacus/tree/develop/feature-libs/checkout",
+  "license": "Apache-2.0",
+  "scripts": {
+    "prepare:schematics": "yarn install",
```

Comment:
We no longer have separate node_modules in feature libs.
The `prepare:schematics` script should be removed along with it's usage. rimraf,tsc, should be referenced from the root node_modules.

Reply:
Fixed.